packages/lesswrong/lib/collections/userRateLimits/permissions.ts

```diff
@@ -0,0 +1,10 @@
+import { adminsGroup } from "../../vulcan-users";
+import { sunshineRegimentGroup } from "../../permissions";
+
+const actions = [
+  "userratelimits.new",
+  "userratelimits.edit.all",
+];
```

Comment:
For bans all of these ones are set for admins:
```
const adminActions = [
  'bans.new',
  'bans.edit.all',
  'bans.remove.all',
  'bans.view.all',
  'bans.remove',
  'bans.edit',
];
```
Is there a reason you aren't setting them all here, are some of them redundant?

Reply:
The only reason is that these were the only ones set on the collection I copied this from :sweat_smile:. Without digging into the code, it seems like these imply some of the others as I tested creating/editing/removing/viewing and all worked fine.